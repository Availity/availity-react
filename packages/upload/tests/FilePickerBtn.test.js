import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { FilePickerBtn } from '..';

afterEach(cleanup);

describe('Upload', () => {
  test('should render', () => {
    const { container } = render(<FilePickerBtn onChange={() => {}} />);

    expect(container).toMatchSnapshot();
  });
});
