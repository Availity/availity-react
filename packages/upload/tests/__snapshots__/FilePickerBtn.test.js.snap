// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Upload should render 1`] = `
<div>
  <div
    class="d-none"
  >
    <input
      class="form-control-file"
      id="filePicker-1"
      type="file"
      value=""
    />
  </div>
  <button
    class="btn btn-primary"
    type="button"
  >
    Select File
  </button>
</div>
`;
